go.mod

```diff
@@ -5,7 +5,7 @@ require (
 	github.com/dustin/go-humanize v1.0.0
 	github.com/go-errors/errors v1.0.1
 	github.com/kballard/go-shellquote v0.0.0-20180428030007-95032a82bc51
-	github.com/mattn/go-isatty v0.0.11
+	github.com/mattn/go-isatty v0.0.16
```

Comment:
Ok, due to the fact that it was introduced with https://github.com/mattn/go-isatty/commit/360bbd2c048732ea365990187a9c3b9304e10f6f in v0.0.13 first.

Reply:
Do you think I should change it to v.0.0.13?  I didn't see that in some list I saw somewhere.

Reply:
Usually I've no problem to update to the latest tag, but above `v0.0.17` we would loose the documented `go` compatibility of `v1.16` due to the dependency of `golang.org/x/sys`. At least as far as I understood.

Reply:
After the update you should squash your last 5 commits and correct the commit description, since you switched from `v0.0.16` to `v0.0.13`.

Reply:
I'm not positive, but I think I've done this.  (I'm not a power user of git.)